packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -427,6 +429,25 @@ Future<XcodeBuildResult> buildXcodeProject({
               return;
             }
           } else {
+            if (!globals.logger.isVerbose) {
+              if (line.contains('error:')) {
+                globals.printError(line);
+              } else if (line.contains('warning:')) {
+                globals.printWarning(line);
+                inWarningBlock = true;
+              } else if (inNoteBlock) {
+                globals.printWarning(line);
+                inNoteBlock = false;
+              } else if (inWarningBlock) {
+                if (line.startsWith(RegExp(r'\s+?note[:]'))) {
+                  // Xcode doesn't echo this, so we don't.
```

Comment:
Xcode doesn't echo notes? I'm pretty sure it does...

Reply:
That "note:" line in particular it doesn't.  I fought with it for a while.  It's a black box.